microsoft-dynamics-365/2.5/modules/ROOT/pages/microsoft-dynamics-365-connector-reference.adoc

```diff
@@ -0,0 +1,989 @@
+* Associate Entity Upon Creation - `+https://msdn.microsoft.com/en-us/library/mt607875.aspx#Anchor_4+`
+* Batch Request - `+https://msdn.microsoft.com/en-us/library/mt607719.aspx+`
+
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Logical Name a| String |  The logical name of the entity. It is always the schema name in lower case. |  | x
+| Attributes a| Object |  The attributes of the entity. |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+| Encoding a| String | The body encoding type | |
+|===
+
+
+=== For Configurations
+
+* <<dynamics-config>>
+
+==== Throws
+
+* DYNAMICS:CONNECTIVITY
+* DYNAMICS:INVALID_PAGE_SIZE
+* DYNAMICS:NOT_FOUND
+* DYNAMICS:OPERATION_FAILED
+* DYNAMICS:PARSE_ERROR
+* DYNAMICS:RETRY_EXHAUSTED
+* DYNAMICS:UNAUTHORIZED
+* DYNAMICS:UNKNOWN
+
+
+[[updateMultiple]]
+=== Update Multiple
+
+`<dynamics:update-multiple>`
+
+
+Update Multiple Entities:
+
+* Update Entity - `+https://msdn.microsoft.com/en-us/library/mt607664.aspx+`
+* Associate Entities Upon Update - `+https://msdn.microsoft.com/en-us/library/mt607875.aspx#Anchor_4+`
+* Batch Request - `+https://msdn.microsoft.com/en-us/library/mt607719.aspx+`
+
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Logical Name a| String |  The logical name of the entity. It is always the schema name in lower case. |  | x
+| Entities Attributes a| Array of Object |  A list with the attributes of the entities that are going to be updated as a List of Maps. |  `#[payload]` |
+| Use Single Transaction a| Boolean |  Use a single transaction for an update. If the update fails, the transaction is rolled back. |  false |
+| Target Variable a| String |  The name of a variable that stores the operation's output |  |
+| Target Value a| String |  An expression that evaluates against the operation's output. The outcome of this expression is stored in the target variable. |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+| Encoding a| String | The body encoding type | |
+|===
+
+==== Output
+
+[%autowidth.spread]
+|===
+|Type |<<BulkOperationResult>>
+|===
+
+=== For Configurations
+
+* <<dynamics-config>>
+
+==== Throws
+
+* DYNAMICS:CONNECTIVITY
+* DYNAMICS:INVALID_PAGE_SIZE
+* DYNAMICS:NOT_FOUND
+* DYNAMICS:OPERATION_FAILED
+* DYNAMICS:PARSE_ERROR
+* DYNAMICS:RETRY_EXHAUSTED
+* DYNAMICS:UNAUTHORIZED
+* DYNAMICS:UNKNOWN
+
+== Sources
+
+* <<on-deleted-object>>
+* <<on-modified-object>>
+* <<on-new-object>>
+
+[[on-deleted-object]]
+=== On Deleted Object
+`<dynamics:on-deleted-object>`
+
+Starts your app when an object is deleted
```

Comment:
```suggestion
Starts your app when an object is deleted.
```

Reply:
I don't agree with adding a period here since it is not a sentence or not in a field description. We can discuss this at a scrum.